osu.Game/Overlays/OnlineOverlay.cs

```diff
@@ -16,10 +17,16 @@ public abstract class OnlineOverlay<T> : FullscreenOverlay<T>
         protected readonly LoadingLayer Loading;
         private readonly Container content;
 
-        protected OnlineOverlay(OverlayColourScheme colourScheme)
+        protected OnlineOverlay(OverlayColourScheme colourScheme, bool requiresSignIn = true)
             : base(colourScheme)
         {
-            base.Content.AddRange(new Drawable[]
+            var mainContent = requiresSignIn
+                ? new OnlineViewContainer($"Sign in to view the {Header.Title.Title}")
```

Comment:
I think what's said in https://github.com/ppy/osu/pull/7546#discussion_r369903295 would still apply here? unless things have changed since then?

Reply:
It's much easier to localise it here because the concatenation isn't split into two places.